### PR TITLE
Fix/deprecated excluded status

### PR DESCRIPTION
**Pull request by fredbi**
## Overview
With opa-utils v0.0.227, a few breaking changes have been introduced:
* the Excluded status has been deprecated.
* exception processing requires controls to be injected
 
This PR adapts kubescape to support these changes, as this block other changes that require an update of opa-utils.

Reports now show 0 for exclusions.
Unit tests that counted exclusions now count these as passed.

cc: @dwertent @amirmalka

### CODE REVIEW

**Comment by amirmalka** [comment]
@fredbi this PR cannot be merged before this PR: https://github.com/kubescape/kubescape/pull/1016

**Comment by fredbi** [comment]
@amirmalka  yes the one you are pointing to doing the compatibility work, so this is redundant with this one.
Maybe it is fine to close my PR.

There is just a small difference: #1016 picks all controls from opap.AllPolicies.Controls (which was my initial pick as well) and this made the unit tests to fail, so I decided to construct the map from opap.Policies.

**Comment by fredbi** [comment]
It so happens that this issue was blocking some other PR of mine, hence the push for a quick fix... :)

**Comment by amirmalka** [comment]
@fredbi I understand that this change is blocking other PRs... we plan on merging it tomorrow. 

cc @dwertent

File: core/pkg/opaprocessor/processorhandlerutils.go [hunk 1]
```diff
@@ -21,21 +21,36 @@ import (
...
 func (opap *OPAProcessor) updateResults(ctx context.Context) {
-	ctx, span := otel.Tracer("").Start(ctx, "OPAProcessor.updateResults")
+	ctx, span := otel.Tracer("").Start(ctx, "OPAstringProcessor.updateResults")
 	defer span.End()
 	// remove data from all objects
 	for i := range opap.AllResources {
 		removeData(opap.AllResources[i])
 	}
 
+	// An exception inherits part of its status from the control
```
Comment (fredbi) [inline core/pkg/opaprocessor/processorhandlerutils.go:31]: not sure about this part

**Pull request closed without merging**